ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivacyController.java

```diff
@@ -65,7 +72,7 @@ public PrivateTransactionHandler(
         privateMarkerTransactionFactory);
   }
 
-  public PrivateTransactionHandler(
+  public PrivacyController(
       final Enclave enclave,
       final String enclavePublicKey,
```

Comment:
Why does this still take a enclavePublicKey at construction rather than on "retrieveTransaction", i.e. isn't that change required for Multitenancy?